doc/content/microreactors/gcmr/GCMR_Core_Neutronics.md

```diff
@@ -0,0 +1,100 @@
+
+!media media/gcmr/core/Fig_5.png
+      style=display: block;margin-left:auto;margin-right:auto;width:80%;
+      id=Fig_5
+      caption= Detailed 3-D GCMR core mesh
+
+
+
+!listing microreactors/gcmr/core/MESH/Griffin_mesh.i
+
+
+
+## Griffin Model
+
+The process initiated with generating homogenized multi-group cross-sections using Serpent-2, which were then converted into XML-format cross-section file. Griffin utilizes the cross-sections in XML-format file in conjunction with the mesh file. The 3D whole-core mesh was constructed utilizing MOOSE’s Reactor module, ensuring meticulous consistency between geometric representations in the mesh file and Serpent-2. Griffin solves the neutron transport equation employing discontinuous finite element DFEM with SN transport and CMFD acceleration, utilizing on-the-fly coarse mesh generation for CMFD. Efforts were dedicated to simplifying the 3D whole-core GC-MR mesh to alleviate computational demands, with careful consideration to avoid excessive mesh sizes, particularly in specific regions like the radial reflector and control drum areas, to ensure proper convergence of DFEM-SN with CMFD. 
+
+
+
+!listing microreactors/gcmr/core/Neutronics/Griffin_steady_state.i
+
+
+## Run Commands
+The Griffin code is a component of the blue_crab code package, and we can execute the simulation as follows:
+
+!listing  
+mpirun -np number_of_cores /path/to/blue_crab-opt -i Griffin_steady_state.i
+
+It should be noticed that the mesh file can be generated using the --mesh-only option as such:
+
+!listing  
+mpirun -np number_of_cores /path/to/blue_crab-opt -i mesh_input.i --mesh-only
+
+## Results
+
+Segmentations of the whole-core GC-MR mesh utilized in the analyses, employing DFEM-SN with CMFD and an 11-energy-group structure, are illustrated in [Fig_6]. The simulation was conducted for 1/6 of the core with reflected boundary conditions on the cut surfaces and vacuum boundaries for the remaining surfaces. The resulting k-eff value for 2 polar angles and 3 azimuthal angles in the SN was determined as 1.051468, reasonably aligning with the k-eff value obtained from the Monte Carlo Serpent-2 code, which is 1.054670 ± 16 pcm. It was determined that even closer alignment could be achieved with higher numbers of polar and azimuthal angles. [Fig_7] presents a comparison between the normalized axial power distribution computed by both Serpent-2 and Griffin for DFEM-SN (2,3) with CMFD.
```

Comment:
DFEM-SN (n_polar=2, n_azimuthal=3) with CMFD*

n case there is confusion about what (2, 3) means

Reply:
Modified.